.travis.yml

```diff
@@ -33,6 +33,25 @@ matrix:
       script:
         - ./clang-format.sh
         - git diff --exit-code
+#adding code to run on power systems
```

Comment:
Not sure the comment adds anything. Can you make this a newline instead?
```suggestion

```

Reply:
Accepted your suggestion